src/App.js

```diff
@@ -1,13 +1,62 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+//data
+const TodoData = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+constructor(){
+  super();
+  this.state = {
+    todos: TodoData,
+    inputText: '' 
+
+
+  }
+}
+
+ handleChange = e => {
+  this.setState({
+    [e.target.name]: e.target.value
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here for brevity:
```javascript
const { name, value } = e.target;
this.setState({ [name]: value });
```